config/initializers/devise.rb

```diff
@@ -251,8 +251,8 @@
   #     mount MyEngine, at: '/my_engine'
   #
   # The router that invoked `devise_for`, in the example above, would be:
-  config.router_name = :basechurch
-  config.parent_controller = 'Basechurch::ApplicationController'
+  # config.router_name = :basechurch
+  config.parent_controller = 'ApplicationController'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.